.gitlab-ci.yml

```diff
@@ -170,60 +150,85 @@ pup5-unit:
   <<: *pup_5
   <<: *unit_tests
 
-pup5.5.7-unit:
-  <<: *pup_5_5_7
-  <<: *unit_tests
-
-pup4.10-unit:
-  <<: *pup_4_10
+pup5.5.10-unit:
+  <<: *pup_5_5_10
   <<: *unit_tests
 
 pup6-unit:
   <<: *pup_6
   <<: *unit_tests
 
-# Acceptance Tests
+# Acceptance tests
 # ==============================================================================
-pup4.10:
-  <<: *pup_4_10
+pup5.5.10:
+  <<: *pup_5_5_10
   <<: *acceptance_base
   script:
     - 'bundle exec rake beaker:suites'
 
-pup4.10-fips:
-  <<: *pup_4_10
+pup5.5.10-fips:
+  <<: *pup_5_5_10
   <<: *acceptance_base
-  <<: *only_with_SIMP_FULL_MATRIX
   script:
     - 'BEAKER_fips=yes bundle exec rake beaker:suites'
 
-pup5.5.7:
-  <<: *pup_5_5_7
+pup5.5.10-no_simp_server:
```

Comment:
This .gitlab-ci.yaml must have been pulled from pupmod-simp-simp, because the suites listed here do *not* exist in stunnel

Reply:
thanks for the explanation. Was getting confused what those were.